mesheryctl/internal/cli/root/system/stop.go

```diff
@@ -247,6 +249,30 @@ func deleteCRD(name string, client *apiextension.Clientset) error {
 	return client.ApiextensionsV1().CustomResourceDefinitions().Delete(context.TODO(), name, metav1.DeleteOptions{})
 }
 
+func deleteNs(ns string, client *kubernetes.Clientset) error {
+	return client.CoreV1().Namespaces().Delete(context.TODO(), ns, metav1.DeleteOptions{})
+}
+
+// checkAbort checks if the `system stop` should be aborted
+func checkAbort() bool {
+	var mesheryDelRes, keepNsRes bool
+	mesheryDelRes = utils.AskForConfirmation("Meshery deployments will be deleted from your cluster. Are you sure you want to continue")
+	if !mesheryDelRes {
+		return true
+	}
+
+	if !utils.KeepNsFlag {
```

Comment:
@panyuenlau,
If the user selects "delete meshery deployments" but "keep the namespace",
In that case, we abort the meshery stop operation. Is that what we want to do ?